engine/src/main/java/com/datatorrent/stram/api/StramEvent.java

```diff
@@ -34,6 +34,7 @@
   private final long id;
   private long timestamp = System.currentTimeMillis();
   private String reason;
+  private LogLevel eventLogLevel;
```

Comment:
we can call it as logLevel, "event" is redundant as class name already suggests that.